test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -596,44 +618,6 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
 
     It "Invoke-WebRequest supports sending XML requests without encoding" {
         $uri = Get-WebListenerUrl -Test POST
-        $body = '<?xml version="1.0"?><foo />'
-        $result = Invoke-WebRequest -Uri $uri -body ([xml]$body) -ContentType 'text/xml' -method 'POST'
-
-        $object = $result.Content | ConvertFrom-Json
-        $object.Data | Should -BeExactly $body
-    }
-
-    It "Invoke-WebRequest supports request that returns page containing CodPage 936 data." {
-        $uri = Get-WebListenerUrl -Test 'Encoding' -TestValue 'CP936'
-        $command = "Invoke-WebRequest -Uri '$uri'"
-
-        $result = ExecuteWebCommand -command $command
-        ValidateResponse -response $result
-
-        $Result.Output.Encoding.CodePage | Should -Be 936
-        $Result.Output.Content | Should -Match '测试123'
-    }
-
-    It "Invoke-WebRequest validate ConnectionTimeoutSeconds option" {
-        $uri = Get-WebListenerUrl -Test 'Delay' -TestValue '5'
-        $command = "Invoke-WebRequest -Uri '$uri' -ConnectionTimeoutSeconds 2"
-
-        $result = ExecuteWebCommand -command $command
-        $result.Error.FullyQualifiedErrorId | Should -Be "ConnectionTimeoutReached,Microsoft.PowerShell.Commands.InvokeWebRequestCommand"
-    }
-
-    It "Invoke-WebRequest validate TimeoutSec alias" {
-        $uri = Get-WebListenerUrl -Test 'Delay' -TestValue '5'
-        $command = "Invoke-WebRequest -Uri '$uri' -TimeoutSec 2"
-
-        $result = ExecuteWebCommand -command $command
-        $result.Error.FullyQualifiedErrorId | Should -Be "ConnectionTimeoutReached,Microsoft.PowerShell.Commands.InvokeWebRequestCommand"
-    }
-
-    It "Validate Invoke-WebRequest error with -Proxy and -NoProxy option" {
-        $uri = Get-WebListenerUrl -Test 'Delay' -TestValue '10'
-        $command = "Invoke-WebRequest -Uri '$uri' -Proxy 'http://127.0.0.1:8080' -NoProxy -TimeoutSec 2"
-
```

Comment:
I wonder why do you remove the tests? Please revert.

Reply:
Definitely didn't mean to! Good catch on this, I must have had a formatter problem.